lib/src/test/kotlin/network/provider/ProviderTest.kt

```diff
@@ -307,10 +334,11 @@ class ProviderTest {
     fun `get declare v3 transaction`() {
         assumeTrue(NetworkConfig.isTestEnabled(requiresGas = false))
 
-        assumeTrue(network == Network.INTEGRATION)
+        assumeTrue(network == Network.GOERLI_INTEGRATION)
         val transactionHash = when (network) {
-            Network.INTEGRATION -> Felt.fromHex("0x86693a36721bb586bee1f8c8b9ea33fbbb7f820dde48d9068dfa94a99ef53")
-            Network.TESTNET -> throw NotImplementedError("No support for testing deploy account v3 on testnet")
+            Network.GOERLI_INTEGRATION -> Felt.fromHex("0x86693a36721bb586bee1f8c8b9ea33fbbb7f820dde48d9068dfa94a99ef53")
+            Network.GOERLI_TESTNET -> throw NotImplementedError("No support for testing deploy account v3 on testnet")
```

Comment:
Could you explain what is the issue here? Is the error message correct?

Reply:
Not sure this is an issue, just that this test is only intended for integration (it's also skipped on other networks via `assume` above). 

At the time of working on #358, goerli testnet did not support Starknet 0.13, so couldn't add v3 tests. Not sure all these tests will be updated, since goerli is deprecated starting 2024. If they will, probably only if it's needed and in a separate PR.

Reply:
Ok thanks. I would suggest changing the error message (even though it is not reached), but consider this as a nitpick.

Reply:
Will be addressed in a follow-up PR: #386